sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,14 +7,14 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>3.1.0-preview</Version>
+		<Version>4.0.0-preview</Version>
```

Comment:
I didn't find breaking changes here, so do you think 3.2.0 is better?

Reply:
I thought that the release of new major APIs for a new release of ARM APIs was a Major release, but it is the same for me... I will change it to 3.2.0

Reply:
Done